spaghetto/src/Spago/Command/Run.purs

```diff
@@ -88,6 +96,23 @@ run = do
             , "'\n\n"
             , "main()"
             ]
+      
+      dependencies <- _.dependencies <$> ask
```

Comment:
You should bundle this one in line 71 where we extract everything else from the environment

```suggestion
```